2013-10-31  Vitaly Katunin  <vkatunin@luxoft.com>

        [GTK] invalid application of 'sizeof' to incomplete type
        https://bugs.webkit.org/show_bug.cgi?id=123502

        Reviewed by NOBODY (OOPS!).

        * Shared/WebCoreArgumentCoders.cpp:

--- a/Source/WebKit2/Shared/WebCoreArgumentCoders.cpp	(revision 158267)
+++ b/Source/WebKit2/Shared/WebCoreArgumentCoders.cpp	(working copy)
@@ -48,6 +48,7 @@
 #include <WebCore/ResourceRequest.h>
 #include <WebCore/ResourceResponse.h>
 #include <WebCore/TextCheckerClient.h>
+#include <WebCore/TransformationMatrix.h>
 #include <WebCore/URL.h>
 #include <WebCore/UserScript.h>
 #include <WebCore/UserStyleSheet.h>
